ocaml/idl/datamodel_types.ml

```diff
@@ -58,38 +58,179 @@ let rel_dundee_plus = "dundee-plus"
 let rel_ely = "ely"
 let rel_falcon = "falcon"
 
-let release_order =
-  [ rel_rio
-  ; rel_miami
-  ; rel_symc
-  ; rel_orlando
-  ; rel_orlando_update_1
-  ; rel_george
-  ; rel_midnight_ride
-  ; rel_cowley
-  ; rel_boston
-  ; rel_tampa
-  ; rel_clearwater
-  ; rel_vgpu_tech_preview
-  ; rel_vgpu_productisation
-  ; rel_clearwater_felton
-  ; rel_clearwater_whetstone
-  ; rel_creedence
-  ; rel_cream
-  ; rel_indigo
-  ; rel_dundee
-  ; rel_dundee_plus
-  ; rel_ely
-  ; rel_falcon
+type api_release = {
+    code_name: string option;
+    version_major: int;
+    version_minor: int;
+    branding: string;
+    xs_branding: string;
+}
+
+let release_order_full = [{
+      code_name     = Some rel_rio;
+      version_major = 1;
+      version_minor = 1;
+      branding      = "XCP 0.5";
```

Comment:
Ah ok, I didn't know rio was wrong, I simply copied it from branding.js. Good to hear it can go, this means we don't need to package 2 docs versions in the server any more.